### ISSUE TITLE
Some general suggestions

### ISSUE THREAD

**Issue by foxpcteam** [nocobase/nocobase#274]
After playing little bit more with nocobase here is what i think that need to be added

Add option to create multi-level / mega menu in the top bar.

Add embed block. This block will to embed external media like images/video, YouTube, Vimeo, iframe etc...

Add option when creating new "collection" to prevent "collection" or specific record for been deleted (only super admin can delete).  
  
Add option when creating new "collection" to be read-only (all the records will be created and will be as read-only) .  
  
Add option to duplicate record (also in bulk action).  
  
Add option to set record as read only (also in bulk action).

**Comment by zhouyanliang** [comment]
Are you using nocobase-next branch?

**Comment by foxpcteam** [comment]
> Are you using nocobase-next branch?

No. But i will start using this branch to see all the new updates.

**Comment by zhouyanliang** [comment]
This branch is under rapid development and we intend to switch it to the main branch at the end of this month. Currently we still have 60 tasks pending in our task list. Also, this branch is still only available in Chinese in many places, we will provide English version soon.
